package.json

```diff
@@ -1,14 +1,12 @@
 {
   "devDependencies": {
-    "jquery": "1.9.1 - 3",
```

Comment:
I appreciate that you were able to get rid of a good amount of plugins here, especially `jquery`. It's crazy how long that plugin has stuck around.